crates/biome_cli/src/commands/mod.rs

```diff
@@ -120,6 +120,11 @@ pub enum BiomeCommand {
             hide_usage
         )]
         organize_imports_enabled: Option<bool>,
+
+        /// Allow to enable or disable the assists.
+        #[bpaf(long("organize-imports-enabled"), argument("true|false"), optional)]
+        assists_enabled: Option<bool>,
```

Comment:
Should this deprecate `--organize-imports-enabled`? Also, should this be this instead?
```suggestion
        #[bpaf(long("assists-enabled"), argument("true|false"), optional)]
        assists_enabled: Option<bool>,
```